### PR TITLE
Prune test data sets

### PR DESCRIPTION
**Pull request by tobim**
With the changes in this PR, the runtime of the unit tests drops to 30 seconds on my iMac, 13 of those are taken by the `consensus` suite.

- [x] Switch from bro conn.log to small_conn.log
- [x] Prune bro dns.log
- [x] Prune bro http.log
- [x] Replace bgpdump updates.txt

Disabled tests from this PR:
- TEST_DISABLED(bro conn log http slices)
- TEST_DISABLED(regression - build an address index from bro events)
- TEST_DISABLED(regression - checking the result single bitmap)
- TEST_DISABLED(regression - manual address bitmap index from bitmaps)
- TEST_DISABLED(regression - manual address bitmap index from 4 byte indexes)
- TEST_DISABLED(regression - bro conn log service http)
- TEST_DISABLED(regression - manual value index for bro conn log service http)

### CODE REVIEW

**Comment by Neverlord** [comment]
It's a bit odd that each log has a different cutoff now. conn log: 20 entries, DNS log: 32 entries, HTTP log: 40 entries, MRT: 100. Is there a particular reason? Otherwise, I'd say we go with one size for all. That way, the code gets simpler (we can verify all sizes at once) and there are fewer things to remember when writing unit tests and the tests become more uniform.

File: libvast/test/column_index.cpp [hunk 1]
```diff
@@ -85,8 +85,8 @@ TEST(bro conn log) {
   for (auto slice : const_bro_conn_log_slices)
     col->add(slice);
   MESSAGE("verify column index");
-  auto pred = unbox(to<predicate>(":addr == 169.254.225.22"));
-  auto expected_result = make_ids({680, 682, 719, 720}, bro_conn_log.size());
+  auto pred = unbox(to<predicate>(":addr == 192.168.1.103"));
+  auto expected_result = make_ids({1, 3, 7, 14, 16}, bro_conn_log.size());
```
Comment (mavam) [inline libvast/test/column_index.cpp]: My main question for this PR would be: how did you check the ground truth? Did you go back to the respective log or did you apply a "code transformation", e.g., to map 680 -> 1, 682 -> 3, etc.

Either is fine, but as the reviewer, I need to be convinced that it is correct.

Comment (mavam) [reply libvast/test/column_index.cpp]: > [..] I gathered the expected values by taking the line numbers - 1 from my editor.

Sounds like a reasonable approach. 👍

**Comment by mavam** [comment]
@tobim I'm getting this error now:
```
../libvast/test/fixtures/events.cpp:200:58: error: no member named 'updates20180124' in namespace 'bgpdump'; did you mean 'updates20140821'?
  bgpdump_txt = inhale<format::bgpdump::reader>(bgpdump::updates20180124);
                                                ~~~~~~~~~^~~~~~~~~~~~~~~
                                                         updates20140821
libvast/test/data.hpp:19:16: note: 'updates20140821' declared here
constexpr auto updates20140821 = "/Users/mavam/code/vast-dev/libvast/test/logs/bgpdump/updates20140821.txt";
               ^
1 error generated.
[10/15] Building CXX object libvast/CMakeFiles/vast-test.dir/test/system/exporter.cpp.o
```

**Comment by mavam** [comment]
There must have been a Jenkins race, because your previous commit went through just fine.

**Comment by mavam** [comment]
Nope, turns out that was my own noobness: I forgot to do a `cmake ..` after a pull. Sorry for the noise.

**Pull request merged**